go/libraries/doltcore/sqle/dprocedures/dolt_checkout.go

```diff
@@ -290,11 +290,15 @@ func checkoutRemoteBranch(ctx *sql.Context, dSess *dsess.DoltSession, dbName str
 	}
 
 	if len(remoteRefs) == 0 {
+		if isTag, err := actions.IsTag(ctx, dbData.Ddb, branchName); err != nil {
+			return "", err
+		} else if isTag && apr.Contains(cli.MoveFlag) {
```

Comment:
Do we need to only execute this logic to when `cli.MoveFlag` is present? My understanding is that we don't document the `MoveFlag`, but the CLI uses it when it calls `dolt_checkout()` to move the working set contents? It seems like we'd want to return the same error message if you tried to check out a tag from the SQL interface, too, right? I may just be missing something on how the `MoveFlag` is used though.